submissions/rocket-silo/rocket-silo.md

```diff
@@ -0,0 +1,21 @@
+
+Ah, the rocket silo. It is the largest in-game entity outside of the crashed spaceship. It is also one of the most expensive craftable entities. However, when its placed it has wierd effects on the electric grid, stranger still if modules are at play. We also mostly know that using productivity modules inside of it pays off. What about around the silo? All of that in this week's Alt-F4!
+
+### Power Usage
+
+To start, lets look at its power usage.It claims it uses at most 4MW (which is true), but it only uses 0.25MW when just manufacturing rocket parts, or a bit more than 3 assembling machine 1s (0.225MW). When does it require 4MW, or a bit more than 51 assembling machine 1s? When the silo doors are moving, or a rocket is being pushed up. Those things must be heavy! It also uses an additional 10 kW idly when its nighttime, but there are presently no surface lights to indicate this. Back in 0.16, however, there were lights on the silo. Stranger still, when modules are applied they increase the energy usage of the silo by more than expected. 4 speed modules inside a silo should make the total energy usage of rocket part making to about 700kW, according to the mathematics behind it. But when observed, its closer to 960kW, or 2 beacons' worth of power. It does not mess with the aforementioned 4MW times like this, only this part. Speaking of modules and beacons...
+
+### Modules and the Silo
+
+You can fit 12 beacons around 3x3 entities, and 16 around oil refineries. You can fit 20 around a silo. 2 beacons with 2 speed module 3s in each is the same as adding a second machine.
+So one silo can do the work of 13 silos, side-by-side. When I mention costs, they are the relevant ores in vanilla without productivity modules in the crafting chain, and I am also using advanced oil processing. Each beacon costs 135 iron, 110 copper, 20 coal, and 411 oil which is quite a bit for an early-game player but nothing if tier 3 modules are automated (https://factoriolab.github.io/list?p=beacon*1&v=1). That's because any tier 3 module is 1,083 iron, 2,000 copper, 190 coal, and 4,129 crude oil (https://factoriolab.github.io/list?p=speed-module-3*1&v=1). While the calculator mentions speed modules specifically, the other modules are equivalent. So 4 tier 3 modules and 2 beacons costs 4,552 iron, 8,270 copper, 800 coal, and 17,334 crude oil (https://factoriolab.github.io/list?p=speed-module-3*4_beacon*2&v=1). That's a lot of raw materials. With certain optimizations, like those shown here https://factorio.com/blog/post/fff-351, its possible to get more out of your module, but the cost still stands. Normally, this cost greatly outmatches the machine. However, there is 1 exception to this rule, and that's the silo. The silo costs a whopping 12,220 iron, 8,600 copper, 437 coal, 11,080 crude oil and 1,000 stone (https://factoriolab.github.io/list?p=rocket-silo*1&v=1). So, its cheaper than making another silo to place down. Less spaghetti involved, too!
```

Comment:
I like this paragraph but is there any way to show the maths of a bit more graphically? Not exactly sure what the best way would be but the long lists feel a bit repetitive. Maybe a table of some sort?